lib/tasks/deploy.rake

```diff
@@ -98,9 +98,11 @@ namespace :deploy do
   # - remote: list
   # - remote: stop
   task :stop do
-    on_each_docker_host do |server|
-      stop_containers(server, defined_service, fetch(:stop_timeout, 30))
+    threads = on_each_docker_host.map do |server|
```

Comment:
Might be missing something, but `on_each_docker_host` is a method expecting a block - are you sure you can call `map` on it?